Library/ViewModels/NoShippingPledgeViewModel.swift

```diff
@@ -997,6 +978,36 @@ public class NoShippingPledgeViewModel: NoShippingPledgeViewModelType, NoShippin
           refTag: refTag
         )
       }
+
+    // MARK: Pledge Over Time
+
+    self.showPledgeOverTimeUI = project.signal
+      .map { ($0.isPledgeOverTimeAllowed ?? false) && featurePledgeOverTimeEnabled() }
+
+    self.pledgeOverTimeConfigData = Signal.combineLatest(
+      self.showPledgeOverTimeUI,
+      project,
+      pledgeTotal
+    )
+    .filter { showPledgeOverTimeUI, _, _ in
+      showPledgeOverTimeUI
+    }
+    .map { _, project, pledgeTotal -> PledgePaymentPlansAndSelectionData in
+      // TODO: temporary code to simulate the ineligible state. Implementation [MBL-1838](https://kickstarter.atlassian.net/browse/MBL-1838)
+      let isIneligible = pledgeTotal < 150
+
+      return PledgePaymentPlansAndSelectionData(
+        selectedPlan: .pledgeInFull,
+        increments: mockPledgePaymentIncrement(),
+        ineligible: isIneligible,
+        project: project
+      )
+    }
+
+    self.pledgeAmountViewHidden = context.map { $0.pledgeAmountViewHidden }
+    self.pledgeAmountSummaryViewHidden = Signal.zip(baseReward, context).map { baseReward, context in
+      (baseReward.isNoReward && context == .update) || context.pledgeAmountSummaryViewHidden
```

Comment:
This could use a code comment explaining it. Why is it hidden for updates with no rewards?

Reply:
Thanks for catching that! I moved the code back to its original position as it was moved by mistake. The change had no impact on functionality, but I’ve ensured it’s now correctly placed.